_includes/merchant-identified-payer.md

```diff
@@ -0,0 +1,17 @@
+## Merchant Identified Payer
```

Comment:
Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)

Reply:
Add an extra newline at the end of the file, please, @marinafrydenb.